modules/rosetta/src/test/scala/org/tessellation/rosetta/domain/construction/ConstructionServiceSuite.scala

```diff
@@ -35,7 +47,10 @@ object ConstructionServiceSuite extends MutableIOSuite with Checkers {
     }
   }
 
-  test("returns InvalidPublicKey when cannot derive public key") { implicit res =>
+  test("returns InvalidPublicKey when cannot derive public key") { res =>
+    implicit val securityProvider = res._1
+    implicit val kryo = res._2
```

Comment:
it can be simpler:
```scala
... { res =>

implicit val (sp, k) = res
```

also, names can be just `sp` and `k` as they are implicitly passed further, but this is up to you

Reply:
Changed.